java/DynamoDBStreamSource/README.md

```diff
@@ -0,0 +1,37 @@
+# DynamoDB Streams Source example
+
+* Flink version: 1.20
+* Flink API: DataStream API
+* Language: Java (11)
+* Flink connectors: DynamoDb Streams Source
+
+
+This example demonstrate how to use Flink DynamoDB Streams source.
```

Comment:
To make it testable, and conform with other example, it should output to KDS

Reply:
done